src/XrdOss/XrdOssConfig.cc

```diff
@@ -2166,6 +2166,7 @@ void XrdOssSys::List_Path(const char *pfx, const char *pname,
          ss += (flags & XRDEXP_PURGE    ? " purge"   : " nopurge");
          ss += (flags & XRDEXP_RCREATE  ? " rcreate" : " norcreate");
          ss += (flags & XRDEXP_STAGE    ? " stage"   : " nostage");
+         ss += (flags & XRDEXP_NOFICL   ? " noficl"  : " ficl");
```

Comment:
Could you write up any new configurable. Nothing fancy, just what you added. and either include it the pull request or send it to me or both.